ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts

```diff
@@ -70,4 +89,87 @@ test.describe('Templates - Template Library', () => {
     await setIgnoreChiralFlagSetting(page, false);
     await placePhenylalanineMustard(page, x + offsetX, y);
   });
+
+  test('Structure Library UI', async ({ page }) => {
+    // Test case: EPMLSOPKET-4265
+    // Overview Templates Library structure
+    await openStructureLibrary(page);
+  });
+
+  test('Open Structure Library tooltip', async ({ page }) => {
+    // Test case: EPMLSOPKET-4265
+    // Verify Structure LIbrary tooltip
+    const button = page.getByTestId('template-lib');
+    await expect(button).toHaveAttribute(
+      'title',
+      'Structure Library (Shift+T)',
+    );
+  });
+
+  test('Template Library', async ({ page }) => {
+    // Test case: EPMLSOPKET-4266
+    // Verify correct display of Template Library
+    const deltaX = 0;
+    const deltaY = 220;
+    const anyX = 638;
+    const anyY = 524;
+    await openStructureLibrary(page);
+    await takeEditorScreenshot(page);
+    await page.mouse.move(anyX, anyY);
+    await page.mouse.wheel(deltaX, deltaY);
+  });
+
+  test(' Functional groups tab', async ({ page }) => {
+    // Test case: EPMLSOPKET-4267
+    // Verify Functional Group tab
+    await openFunctionalGroup(page);
+  });
+
+  test(' Functional groups - adding structure', async ({ page }) => {
+    // Test case: EPMLSOPKET-4267
+    // Add sturucture from Functional Group into canvas
+    await selectFunctionalGroups(FunctionalGroups.FMOC, page);
+    await clickInTheMiddleOfTheScreen(page);
+  });
+
+  test('Edit templates', async ({ page }) => {
+    // Test case: EPMLSOPKET-1699
+    // Verify correct display of Template Edit window
+    await editStructureTemplate(page, 'β-D-Sugars', 'β-D-Allopyranose');
+  });
+
+  test('Edit templates - name field with no character', async ({ page }) => {
+    // Test case: EPMLSOPKET-1699
+    // Verify validation if name field not contain any characters
+    await editStructureTemplate(page, 'β-D-Sugars', 'β-D-Allopyranose');
+    await page.getByTestId('file-name-input').click();
+    await page.keyboard.press('Control+a');
+    await page.keyboard.press('Delete');
```

Comment:
I believe we can move this code to separate function because it repeates 3 times at least

Reply:
Also I would propose to make structure template name as a parameter of this new function